### PR TITLE
Feature: Add new "Add Teacher" option

### PR DESCRIPTION
**Pull request by danialazmie**
## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Comment by sorogema** [comment]
please add more commenting

File: main.py [hunk 1]
```diff
@@ -39,13 +39,13 @@ def main_menu():
         elif choice == 4:
             search_student()
 
-        if choice == 1:
+        if choice == 5:
             add_teacher()
-        elif choice == 2:
```
Comment (sorogema) [inline main.py:44]: need more clarification on this

**Review by sorogema — COMMENTED** [review]
Please see main.py comment

File: main.py [hunk 2]
```diff
@@ -39,13 +39,13 @@ def main_menu():
...
+        elif choice == 6:
```
Comment (danialazmie) [inline main.py:44]: how about this line

**Pull request merged**